packages/transformers/markdown/src/MarkedTransformer.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+import {Transformer} from '@parcel/plugin';
+import {promisify} from '@parcel/utils';
+import marked from 'marked';
+import hljs from 'highlight.js';
+import fm from 'front-matter';
+import Mustache from 'mustache';
+
+const DEFAULT_TEMPLATE = `<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
```

Comment:
Is there a way to include css from node_modules into html in Parcel 2?

Either wanna import a css file that has some basic styling for the markdown or just import highlight.js's github theme. For both I'd need that functionality, I can always hack it by inlining it but would prefer to do it properly